src/eei.cpp

```diff
@@ -578,11 +646,16 @@ namespace HeraVM {
       create_message.flags = 0;
 
       evm_result create_result;
+     
+      takeGas(create_message.gas);
+      takeGas(GasSchedule::create);
+      takeGas(GasSchedule::callnewacct);
       context->fn_table->call(&create_result, context, &create_message);
-      if (create_result.status_code == EVM_SUCCESS) {
+
+      if (create_result.status_code == EVM_SUCCESS)
         storeUint160(create_result.create_address, resultOffset);
-        lastReturnData.clear();
-      } else if (create_result.output_data) {
```

Comment:
Please don't change this.

Reply:
just fixed this, leftover trash from a rebase